CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLWorker.cc

```diff
@@ -845,7 +881,7 @@ void SiPixelLorentzAnglePCLWorker::fillDescriptions(edm::ConfigurationDescriptio
   desc.add<double>("clustChargeMaxPerLength", 50000)
       ->setComment("maximum cluster charge per unit length of pixel depth (z)");
   desc.add<int>("binsDepth", 50)->setComment("# bins for electron production depth axis");
-  desc.add<int>("binsDrift", 200)->setComment("# bins for electron drift axis");
+  desc.add<int>("binsDrift", 100)->setComment("# bins for electron drift axis");
```

Comment:
This is not modified in CalibTracker/SiPixelLorentzAngle/python/SiPixelLorentzAnglePCLWorker_cfi.py, and therefore this change won't have ay practical effect on the actual configs. Is it intended?

Reply:
done at https://github.com/cms-sw/cmssw/pull/36565/commits/3b421bc3a014a8c63325bc4c6448220c2d073621